src/resources/custom-fields-resource.js

```diff
@@ -1,24 +1,18 @@
-export default function CustomFieldsResource({apiHandler}) {
-    return {
-        getAll({resource, limit = null, offset = null, fields = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                fields,
-            };
-            return apiHandler.getAll(`custom-fields/${resource}`, params);
-        },
-
-        get({resource, name}) {
-            return apiHandler.get(`custom-fields/${resource}/${name}`);
-        },
+/**
+* This file was auto-generated by rebilly-sdk-generator.
+* Do not make direct changes to this file.
+*/
 
-        create({resource, name, data}) {
-            return apiHandler.put(`custom-fields/${resource}/${name}`, data);
-        },
-
-        update({resource, name, data}) {
-            return apiHandler.put(`custom-fields/${resource}/${name}`, data);
-        },
-    };
-};
+export default function CustomFieldsResource({apiHandler}) {
+  return {
+    getAll({resource}) {
+      return apiHandler.getAll(`custom-fields/${resource}`);
+    },
```

Comment:
Is it ok that `getAll` no longer adds query parameters?

Reply:
I think so. Those old parameters were not defined in the backend.